.jshintrc

```diff
@@ -17,7 +17,8 @@
   "nonbsp": true,
   "maxdepth": 8,
   "quotmark": true, // deprecated
-
+  "-W041": false,
+  
   /* relaxing options */
```

Comment:
@mickhansen should we allow comments inside json structure, I know @philip1986 is adding comment because there were some comments in `.jshintrc` already, but I think we should get rid of all comments inside any `json` structure.